p2panda-core/Cargo.toml

```diff
@@ -22,4 +22,5 @@ serde_bytes = { version = "0.11.15" }
 thiserror = "1.0.61"
 
 [dev-dependencies]
+anyhow = "1.0.86"
```

Comment:
I've removed this dependency here as we're not handling any errors in the examples